src/python/test/test_cubical_complex.py

```diff
@@ -147,3 +147,8 @@ def test_connected_sublevel_sets():
                                  periodic_dimensions = periodic_dimensions)
     assert cub.persistence() == [(0, (2.0, float("inf")))]
     assert cub.betti_numbers() == [1, 0, 0]
+
+def test_connected_sublevel_sets():
+    cub = CubicalComplex(top_dimensional_cells = [[0, 0, 0], [0, 1, 0], [0, 0, 0]])
+    cub.persistence()
+    assert cub.cofaces_of_persistence_pairs() == np.array([[1, 7, 4], [0, 8, -1]])
```

Comment:
This test fails in the continuous integration. You probably want to use `np.array_equal` for that.

Reply:
Yes, thanks. Fixed it.